sync/http/alive.go

```diff
@@ -0,0 +1,37 @@
+package http
+
+import (
+	"net/http"
+)
+
+// AliveStatus type representing the liveness of the service via HTTP component.
+type AliveStatus int
+
+const (
+	// Initializing represents a state when warming up and before the component is Alive.
+	Initializing AliveStatus = 0
```

Comment:
The liveness has 2 states: alive (HTTP 200 and not HTTP 500). We do not need a third state anymore.